### PR TITLE
add musl and libressl images, along with a lynx container

### PR DESCRIPTION
**Pull request by azimut**
initial support for #87

there are probably things to clean up, like I am using the `DEFAULT_BUILDER` hack to fix the dependencies but it should be good enough to ship it. To test it you can build it and try it might be using the **lynx** example and changing the parent image. I do use some of the other combinations but I don't know how well they will adapt to this namespace.

### CODE REVIEW

**Comment by edannenberg** [comment]
Thanks for this! Gave it a quick test and looks good to me.

I have added a fix for the `builder_commit_id` issue so you can rebase your `musllibressl` branch on current master and remove the workaround then `git push --force`.

Some minor stuff:

* Not sure if intended but help is not working in Lynx, probably something users would like to have
* This PR should not include any `PACKAGES.md` files, instead they will be added with the next monthly rebuild commit

File: dock/kubler/images/libressl-musl/build.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    echo 'USE="${USE} libressl"'   >> /etc/portage/make.conf 
+    echo "-libressl"               >> /etc/portage/profile/use.stable.mask 
+    echo "-curl_ssl_libressl"      >> /etc/portage/profile/use.stable.mask 
+    echo "dev-libs/openssl"        >> /etc/portage/package.mask/openssl
+    echo "=dev-libs/libressl-2.4*" >> /etc/portage/package.accept_keywords/libressl
+    emerge -f libressl 
+    emerge -C openssh
+    emerge -C openssl 
+    emerge -1v libressl
+    emerge -1q wget
```
Comment (jbergstroem) [inline dock/kubler/images/libressl-musl/build.sh]: The use of `-q` seems fairly inconsistent. Is there a reason why?

Comment (azimut) [inline dock/kubler/images/libressl-musl/build.sh]: I think they are from a way back when I was troubleshooting this. I updated the emerge commands to use -q as on the wiki.

**Comment by azimut** [comment]
@edannenberg 
```
Not sure if intended but help is not working in Lynx, probably something users would like to have
```
Cannot reproduce, this works for me:
```
[~] > sudo docker run --rm -ti --name lynx kubler/lynx  --help
```

**Comment by edannenberg** [comment]
@azimut I meant the help inside lynx, only used w3m as console browser, so first thing I hit on the keyboard was h :P

**Comment by azimut** [comment]
@edannenberg oh, looks like the help files are at /usr/share/doc/lynx-2.8.9_pre11/ but those don't get installed on ROOT only on the builder. I can install it on the builder and copy it over. But do you know of other way to temporary disable that from emerge?

**Comment by azimut** [comment]
ah, FEATURES nodoc, got it

**Comment by edannenberg** [comment]
:+1: Will merge this weekend.

**Comment by edannenberg** [comment]
Added some missing `README.md` files and gave it a bit more polishing. Sqashed and merged manually. :shipit:

FYI: Removed the `-q` emerge flag and made `--quiet-build` default for all emerge calls, passing --verbose-build (or -v) to the build command reverts to the old behaviour.

**Pull request closed without merging**